components/menu/MenuItem.tsx

```diff
@@ -54,50 +38,47 @@ export const MenuItem: FC<MenuItemProps> = observer(
     open,
     onClick,
   }) => {
-    const store = useStore();
-
-    const classes = useMemo(() => {
-      return `flex flex-row h-ztg-56 items-center cursor-pointer
-      ${
-        store.leftDrawerClosed === true
-          ? "justify-center"
-          : "py-ztg-16 px-ztg-11"
-      }
-    `;
-    }, [store.leftDrawerClosed]);
+    const { pathname } = useRouter();
+    const [isHovered, setIsHovered] = useState(false);
+    active = pathname === "/" ? false : active;
 
     return (
       <WrapComponent href={href}>
         <div
-          className={`${classes} ${className} ${
-            active
-              ? "bg-border-dark text-white dark:bg-sky-1100 font-bold"
-              : "ztg-transition text-sky-600 hover:text-white"
-          } 
-           rounded-ztg-10 
-          mx-ztg-20
-          `}
+          className={`flex rounded-full p-5 w-[68px] ${className}`}
+          onMouseEnter={() => setIsHovered(true)}
+          onMouseLeave={() => setIsHovered(false)}
           onClick={onClick}
+          style={{
+            border: isHovered
```

Comment:
Can we do this will css instead? Whilst the js is loading the menu isn't functional with this approach. Tailwinds `hover:` should work

Reply:
I didn't think I could initially, but then I saw that tailwind has something called group effects for child elements so I was able to refactor it

Reply:
I discovered Tailwind had something called [group effects baed on parent/sibling states](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state) so I was able to refactor it.

Reply:
oh that's nice, didn't know about that either